pom.xml

```diff
@@ -262,8 +262,28 @@
                         <failOnMissingWebXml>false</failOnMissingWebXml>
                     </configuration>
                 </plugin>
+
             </plugins>
         </pluginManagement>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <executions>
```

Comment:
maybe good to add a version to this plugin? 
this is the one we used in other project 
```
<version>2.9</version>
```